docs/content/en/docs/getting-started/vsphere/vsphere-getstarted.md

```diff
@@ -108,13 +108,20 @@ Make sure you use single quotes around the values so that your shell does not in
    {{% /alert %}}
 
      
-1. Create cluster
+1. Create cluster  
+   {{% alert title="Important" color="warning" %}}
+  
+   If your vSphere version is 7.0.2, then few user privileges don't exist
```

Comment:
Is this true for all vSphere versions 7.0 and above?

Reply:
@csplinter Do you foresee any changes or the PR's good to merge? Just checking here if this can be closed